Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[workspace]
+members = [
+    "cli",
+    "lib"
+]
+
+[workspace.package]
+version = "0.1.0"
+authors = ["Aarav Mehta <aarav05mehta@gmail.com> (Google Summer of Code Participant)", "Pavel Nikonorov <pavel@genxt.network> (Google Summer of Code Mentor; GENXT)", "ELIXIR Cloud & AAI <cloud-service@elixir-europe.org> (ELIXIR Europe)"]
```

Comment:
I would not include the roles here (e.g., "Google Summer of Code Participant"). If you really want to add such information, I think a `contributors.md` file in the root directory would be the most appropriate place.